### DISCUSSION TITLE
返回对象里如果有个时间戳Long类型 会变成字符串是咋回事

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#269] [02.提问] [answered]
<img width="757" alt="image" src="https://user-images.githubusercontent.com/6092525/230565339-61ae4bee-ec4e-4c92-a29c-fcff28deb061.png">
<img width="279" alt="image" src="https://user-images.githubusercontent.com/6092525/230565492-4df5b413-b0c7-4b9d-ae62-33890a7d6897.png">

**Comment by zuihou** [comment, marked as answer]
防止精度丢失。

<img width="1276" alt="image" src="https://user-images.githubusercontent.com/8500410/230566001-d51bd15e-897d-414e-91de-9fa917b54d67.png">

<img width="1262" alt="image" src="https://user-images.githubusercontent.com/8500410/230566050-c0108371-bd81-4983-a834-dc3d42331a1f.png">

**Comment by zuihou** [comment]
https://cloud.tencent.com/developer/article/1703457
